wire-compiler/src/main/java/com/squareup/wire/schema/WireRun.kt

```diff
@@ -225,3 +225,7 @@ data class WireRun(
     return result
   }
 }
+
+fun List<ProtoFile>.link(): Schema {
```

Comment:
Agree with Jake's comments, but it also confuses me why we want this method to live in `WireRun.kt`. It should either be a part of `Schema.java`, or maybe `NewSchemaLoader`.

Reply:
moved to Schema as a public static method